modules/vmware-csi-driver-reqs.adoc

```diff
@@ -21,3 +21,133 @@ If a third-party vSphere CSI driver is present in the cluster, {product-title} d
 ====
 The VMware vSphere CSI Driver Operator is supported only on clusters deployed with `platform: vsphere` in the installation manifest.
 ====
+
+You can create a custom role for the Container Storage Interface (CSI) and the CSI Driver Operator. The custom role can include privilege sets that assign a minimum set of permissions to each vSphere object. This means that the Container Storage Interface (CSI) and CSI Driver Operator can establish a basic interaction with these objects. The following tables detail these minimum set of permissions.
+
+[IMPORTANT]
+====
+Installing an {product-title} cluster in a vCenter is tested against a full list of privileges as described in the "Required vCenter account privileges" section. By adhering to the full list of privileges, you can reduce the possibility of unexpected and unsupported behaviors that might occur when creating a custom role with a set of restricted privileges. 
+====
+
+.Minimum permissions for the CSI
```

Comment:
@dfitzmau @rvanderp3 Can we use a "Storage" which contains `CSI Driver` +  `CSI Driver Operator` + `vsphere problem detector` (we missed here)? Does it make more sense? Because we can disable "Storage" capability but not for each of them, and it would make things simple.

Reply:
Hi @duanwei33 . I merged these tables into 1 table. WDYT?